src/vs/base/parts/storage/test/node/storage.test.ts

```diff
@@ -206,7 +206,8 @@ suite('Storage Library', () => {
 		await rimraf(storageDir, RimRafMode.MOVE);
 	});
 
-	test('conflicting updates', async () => {
+	// test is disabled due to failures
+	test('conflicting updates @UNSTABLE@', async () => {
```

Comment:
This tag wont work here. you need to mark it as skip. I think its just test.skip(

@Charles-Gagnon has done that before.

Reply:
Yup, just change it to `test.skip('conflicting updates...`